LoginItemsAE.c

```diff
@@ -94,7 +94,7 @@ static OSStatus LaunchSystemEvents(ProcessSerialNumber *psnPtr)
     // Launch it!
     
     if (err == noErr) {
-        if ( LSOpenApplication != NULL ) {
+        if ( &LSOpenApplication != NULL ) {
```

Comment:
I don't have time to look into it right now, but this seems possibly incorrect? Then again, I didn't write this code, so maybe it's fine.